### PR TITLE
Add link to record detail view

### PR DESCRIPTION
**Pull request by raivop**

### CODE REVIEW

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy [hunk 1]
```diff
@@ -603,7 +603,17 @@ class OccurrenceTagLib {
...
                         td(class: 'search-results-cell', *:properties) {
                             try {
-                                mkp.yieldUnescaped(formatValue(occurrence.get(column), column))
+                                def value = occurrence.get(column)
+
+                                if(column == 'scientificName') {
+                                    // Record view link.
+                                    a(
+                                        href: g.createLink(url:"${request.contextPath}/occurrences/${occurrence.uuid}"),
+                                        value ? value : g.message(code:'formatListRecordRow.viewRecord')
```
Comment (timo-piirmann) [inline grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: space after ":" on both lines would be nice

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy [hunk 2]
```diff
@@ -603,7 +603,17 @@ class OccurrenceTagLib {
 
                         td(class: 'search-results-cell', *:properties) {
                             try {
-                                mkp.yieldUnescaped(formatValue(occurrence.get(column), column))
+                                def value = occurrence.get(column)
+
+                                if(column == 'scientificName') {
```
Comment (timo-piirmann) [inline grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: And maybe a constant '' or "" use

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy (hunk 1 above)
Comment (raivop) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: Ok, note to self: never simply copy anything that exists in that code.

File: grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy (hunk 2 above)
Comment (raivop) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: Single quotes there are on purpose. It's simple string.

Comment (timo-piirmann) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: Why would anyone differentiate such a thing?

Comment (raivop) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: In groovy, you can do different things with single and double quote strings.

Comment (timo-piirmann) [reply grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy]: Double quotes can do everything single quotes can and more so there's really 0 reason to use single quotes. But oh well

**Pull request merged**